lib/rspec/core/output_wrapper.rb

```diff
@@ -3,17 +3,25 @@ module Core
     # @private
     class OutputWrapper
       # @private
-      attr_writer :output
+      attr_accessor :output
 
       # @private
       def initialize(output)
         @output = output
       end
 
+      def respond_to?(name, priv=false)
+        output.respond_to?(name, priv)
+      end
+
+      def method_missing(name, *args, &block)
+        output.method_missing(name, *args, &block)
+      end
```

Comment:
Ah, I thought we didn't define `method_missing` here.  What's the reason to do that?  Do we need to proxy any methods that are not part of the `IO.instance_methods` interface?  (I thought we didn't).